doc/source/main-documentation/qibolab.rst

```diff
@@ -68,10 +68,11 @@ Now we can create a simple sequence (again, without explicitly giving any qubit
 
    ps = PulseSequence()
    qubit = platform.qubits[0]
-   ps.concatenate(qubit.native_gates.RX.create_sequence())
-   ps.concatenate(qubit.native_gates.RX.create_sequence(phi=np.pi / 2))
+   natives = platform.parameters.native_gates.single_qubit[0]
```

Comment:
If I understand correctly, this is the new interface for accessing native gates. Would it be a bit more user-friendly to make the trivial
```py
@property
def native_gates(self):
    return self.parameters.native_gates
```
in the `Platform`, so that users can use `platform.native_gates` directly? This is having in mine users who may not care to know the internal structure of the `Platform` or `parameters` but still want to access native gates.

Reply:
Definitely. I spent no time thinking about the interface.

I have nothing against the shortcut.

Reply:
Done in 8b2c7d8d37b6803b9825d614b9437570bee90b4c